regression/Makefile

```diff
@@ -20,15 +20,37 @@ DIRS = ansi-c \
        test-script \
        # Empty last line
 
-# Check for the existence of $dir. Tests under goto-gcc cannot be run on
-# Windows, so appveyor.yml unlinks the entire directory under Windows.
+# Run all test directories in sequence
+.PHONY: test
 test:
 	@for dir in $(DIRS); do \
```

Comment:
How  about `@for dir in $(wildcard $(DIRS)); do` to have make filter out non-existent directories?

Reply:
Thanks for that suggestion, I can make things a lot simpler with that idea.